bin/rbenv-update

```diff
@@ -26,33 +27,57 @@ else
   reset=""
 fi
 
-indent_output() {
-  while read data; do
-    printf " ${color}|${reset}  %s\n" "$data"
-  done
+# Helper functions
+pretty_output() {
+  if [[ -n "$verbose" ]]; then
+    while read -r data; do
+      printf " ${color}|${reset}  %s\n" "$data"
+    done
+  else
+    cat >/dev/null 2>&1
```

Comment:
Since this function will have things piped to it, this is here to prevent blocking.